server/public/model/post.go

```diff
@@ -19,40 +19,39 @@ import (
 )
 
 const (
-	PostSystemMessagePrefix        = "system_"
-	PostTypeDefault                = ""
-	PostTypeSlackAttachment        = "slack_attachment"
-	PostTypeSystemGeneric          = "system_generic"
-	PostTypeJoinLeave              = "system_join_leave" // Deprecated, use PostJoinChannel or PostLeaveChannel instead
-	PostTypeJoinChannel            = "system_join_channel"
-	PostTypeGuestJoinChannel       = "system_guest_join_channel"
-	PostTypeLeaveChannel           = "system_leave_channel"
-	PostTypeJoinTeam               = "system_join_team"
-	PostTypeLeaveTeam              = "system_leave_team"
-	PostTypeAutoResponder          = "system_auto_responder"
-	PostTypeAddRemove              = "system_add_remove" // Deprecated, use PostAddToChannel or PostRemoveFromChannel instead
-	PostTypeAddToChannel           = "system_add_to_channel"
-	PostTypeAddGuestToChannel      = "system_add_guest_to_chan"
-	PostTypeRemoveFromChannel      = "system_remove_from_channel"
-	PostTypeMoveChannel            = "system_move_channel"
-	PostTypeAddToTeam              = "system_add_to_team"
-	PostTypeRemoveFromTeam         = "system_remove_from_team"
-	PostTypeHeaderChange           = "system_header_change"
-	PostTypeDisplaynameChange      = "system_displayname_change"
-	PostTypeConvertChannel         = "system_convert_channel"
-	PostTypePurposeChange          = "system_purpose_change"
-	PostTypeChannelDeleted         = "system_channel_deleted"
-	PostTypeChannelRestored        = "system_channel_restored"
-	PostTypeEphemeral              = "system_ephemeral"
-	PostTypeChangeChannelPrivacy   = "system_change_chan_privacy"
-	PostTypeWrangler               = "system_wrangler"
-	PostTypeGMConvertedToChannel   = "system_gm_to_channel"
-	PostTypeAddBotTeamsChannels    = "add_bot_teams_channels"
-	PostTypeSystemWarnMetricStatus = "warn_metric_status"
-	PostTypeMe                     = "me"
-	PostCustomTypePrefix           = "custom_"
-	PostTypeReminder               = "reminder"
-	PostTypeAdminReport            = "system_admin_report"
```

Comment:
Only removed here `PostTypeSystemWarnMetricStatus`, diff is having a hard time showing just that line change